docker-compose.yml

```diff
@@ -1,58 +1,51 @@
-# Use admin/pass as user/password credentials to login to openemr (from OE_USER and OE_PASS below)
-# MYSQL_HOST and MYSQL_ROOT_PASS are required for openemr
-# FLEX_REPOSITORY and (FLEX_REPOSITORY_BRANCH or FLEX_REPOSITORY_TAG) are required for flex openemr
-# EASY_DEV_MODE prevents issues with permissions when mounting volumes
-# MYSQL_USER, MYSQL_PASS, OE_USER, MYSQL_PASS are optional for openemr and
-#   if not provided, then default to openemr, openemr, admin, and pass respectively.
 version: '3.1'
 services:
   mysql:
     restart: always
-    image: mariadb:10.4
-    command: ['mysqld','--character-set-server=utf8']
+    image: 'mariadb:10.4'
+    command:
+      - mysqld
+      - '--character-set-server=utf8'
```

Comment:
hi @ophthal , just checking if all these changes in this docker-compose.yml file were required for it to work?

Reply:
No.  THey were copied so I could run it on my local device in a docker container.  Feel free to gitignore those.